app/services/usersService.js

```diff
@@ -0,0 +1,29 @@
+const { User } = require('../models');
+const { notFoundError } = require('../errors');
+const { userNotFoundErrorMessage, getUserFields } = require('../helpers');
+const { port, host, pageSize } = require('../../config').common.api;
+
+exports.getAllUsers = (getPage = 1) => () => {
+  const page = parseInt(getPage);
+  const offset = (page - 1) * pageSize;
+  const limit = parseInt(pageSize);
+
+  return User.getAll({ offset, limit }).then(response => {
+    const areNext = offset + limit < response.count;
+
+    return {
+      count: response.count,
```

Comment:
You could be a serializer for this response, in that way if you do it generic another findAll should be easy to build